modules/ROOT/pages/bare-metal.adoc

```diff
@@ -113,3 +117,73 @@ IPAPPEND 2
 == PXE rootfs image
 
 include::pxe-artifacts.adoc[]
+
+== Automated ISO/PXE installs with Ignition embedding
+
+The Fedora CoreOS Live environment is also CoreOS in the sense that it can boot via Ignition, execute containers, etc.  It includes all of the same content.
+
+As noted above for Live PXE, the live environment (including ISO) does not have to actually perform 
+a persistent installation.  You can boot it from a read-only medium (physical ISO, USB stick with a physical read-only switch),
```

Comment:
Hmm, is the "read-only" and "physical read-only switch" part more confusing than it's worth?  The live environment works just as well from r/w media.